src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't apply any rules to the * selector except box-sizing

Reply:
ok